debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java

```diff
@@ -85,6 +86,7 @@ public class PostgresConnection extends JdbcConnection {
 
     private final TypeRegistry typeRegistry;
     private final PostgresDefaultValueConverter defaultValueConverter;
+    private final JdbcConfiguration jdbcConfig;
```

Comment:
why is this new instance variable required ?

Reply:
This is required as we access the hostname using `jdbcConfig.hostname()` in the helper methods of this class.